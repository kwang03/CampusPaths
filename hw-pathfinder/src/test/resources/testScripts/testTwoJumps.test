#Tests Dijkstra's when the least cost path has two edges

CreateGraph myGraph

AddNode myGraph A
AddNode myGraph B
AddNode myGraph C
AddNode myGraph D

AddEdge myGraph A B 1.0
AddEdge myGraph A D 1.0
AddEdge myGraph B C 1.0
AddEdge myGraph D C 2.0

FindPath myGraph A C