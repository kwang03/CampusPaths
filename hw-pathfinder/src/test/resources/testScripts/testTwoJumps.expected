#Tests Dijkstra's when the least cost path has two edges

created graph myGraph

added node A to myGraph
added node B to myGraph
added node C to myGraph
added node D to myGraph

added edge 1.000 from A to B in myGraph
added edge 1.000 from A to D in myGraph
added edge 1.000 from B to C in myGraph
added edge 2.000 from D to C in myGraph

path from A to C:
A to B with weight 1.000
B to C with weight 1.000
total cost: 2.000